openpifpaf/visualizer/caf.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, meta: headmeta.Caf):
         super().__init__(meta.name)
         self.meta = meta
         keypoint_painter = show.KeypointPainter(monocolor_connections=True)
-        self.annotation_painter = show.AnnotationPainter(keypoint_painter=keypoint_painter)
+        self.annotation_painter = show.AnnotationPainter(
+                                    painters={'Annotation': keypoint_painter})
```

Comment:
Throughout the code there are only two types of indent: indent by one level (4 spaces) or hanging indent (align with a parenthesis). Can you please fix here?